TACC/Core-CMS-Resources
# FP-1217: New UTRC Logo

# Overview & Changes

- Added new UTRC logo.
- Restored default branding for UTRC.


# Issues

- [FP-1217](https://jira.tacc.utexas.edu/browse/FP-1217)

# Screenshots
![UTRC New Logo](https://user-images.githubusercontent.com/62723358/134395539-b1225df0-6dcb-441c-868d-3987d759db1a.png)

# Testing

1. Load/Deploy a UTRC CMS with this branch.
2. See the new logo, "🌐 UTRC".
3. See branding is "TACC | 🛡TEXAS".
